src/test/java/br/com/moip/api/InvoiceAPITest.java

```diff
@@ -53,15 +51,18 @@ public void testCreate() {
                                                 new int[]{6, 9})
                                         .supressShippingAddress(true)));
 
-        assertEquals("INV-7761BDB06412", invoiceCreated.getId());
+        assertEquals("INV-B36128A22351", invoiceCreated.getId());
+        assertEquals(Integer.valueOf(1000), invoiceCreated.getAmount().getTotal());
+        assertEquals("My Invoice Product", invoiceCreated.getItems().get(0).getProduct());
```

Comment:
The product and detail asserts are ok? Because you create invoice with a different text...